kpatch-build/create-diff-object.c

```diff
@@ -214,6 +216,22 @@ static struct rela *toc_rela(const struct rela *rela)
 				   (unsigned int)rela->addend);
 }
 
+#ifdef __aarch64__
+static int kpatch_is_mapping_symbol(struct symbol *sym)
```

Comment:
This could use a comment to describe what a mapping symbol is and why we wouldn't correlate it.

Reply:
Also, probably best to make it return a bool so the return code semantics are clear.

Reply:
Yep, noted